libfwupdplugin/fu-device.h

```diff
@@ -640,6 +640,15 @@ fu_device_new(FuContext *ctx);
  */
 #define FU_DEVICE_PRIVATE_FLAG_SKIPS_RESTART "skips-restart"
 
+/**
+ * FU_DEVICE_PRIVATE_FLAG_IS_FAKE:
+ *
+ * The device is fake, and constructed from a fake sysfs test directory.
```

Comment:
can't use emulated instead?

Reply:
Exactly that -- i.e. we create them enough for them to be added to the daemon as valid devices, but we can't interact with them in any way.